lib/codit/outbreak.py

```diff
@@ -75,14 +81,20 @@ def plot(self, **kwargs):
 
 
 class OutbreakRecorder:
-    def __init__(self):
+    def __init__(self, show_heatmap=False):
         self.story = []
         self.realized_r0 = None
+        self.show_heatmap = show_heatmap
+        self.visualiser = None
+
 
     def record_step(self, o):
         N = len(o.pop.people)
         # pot_haz = sum([covid_hazard(person.age) for person in o.pop.people])
         # tot_haz = sum([covid_hazard(person.age) for person in o.pop.infected()])
+        if self.show_heatmap and o.step_num == 1:
```

Comment:
Should ths be `o.step_num == 0`? Or should the visualiser be created before the loop begins?

Reply:
self.update_time() is before the self.record_state() in simulate, so the first o.step_num has beed changed to 1 in self.update_time(). So o.step_num is never going to be 0.

Reply:
self.update_time() is before the self.record_state() in simulate, so the first o.step_num has beed changed to 1 in self.update_time(). So o.step_num is never going to be 0.